ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -405,7 +405,18 @@ def __init__(self, log_dir):
             raise RuntimeError("This contrib module requires tensorboardX to be installed. "
                                "Please install it with command: \n pip install tensorboardX")
 
-        self.writer = SummaryWriter(logdir=log_dir)
+        try:
+            self.writer = SummaryWriter(logdir=log_dir)
+        except TypeError:
+            from pkg_resources import get_distribution
```

Comment:
Should we install `setuptools` to have this package ?
Can't we just call
```
import tensorboardX
if tensorboardX.__version__ < "1.7":
    ...
```

Reply:
I tried that, and for some reason `tensorboardX` doesn't expose a `__version__` attribute.

Reply:
That's true : https://github.com/lanpa/tensorboardX/commit/28cbf7f5a82067662f0f983080ee30baa0411ca2#commitcomment-33048905
and this was fixed in `1.7`.

Reply:
Then we could use the attribute error to infer that the version < 1.7, which is kind of ironic (the lack of a version gives us the version).  That would let us remove the `pkg_resources` import - you want me to commit that?

Reply:
Yes, please! That was my suggestion below, actually :)